bin/dotd

```diff
@@ -291,27 +291,20 @@ def do_dtt_subroutine
   print_dtt_message
   should_i 'DTT' do
     view_commits base: 'test', head: 'staging'
-    should_i 'merge from staging to test' do
-      should_i 'reset DTT message to in progress' do
-        DevelopersTopic.set_dtt "in progress (@#{@dotd_name})"
-      end
-      pr_number = GitHub.create_and_merge_pull_request(
-        base: 'test',
-        head: 'staging',
-        title: 'DTT (Staging > Test)'
-      )
-      url = GitHub.url(pr_number)
-      wait_for "DTT (#{url}) to complete, re-run / investigate failures"
 
-      should_i 'reset DTT message to yes' do
-        DevelopersTopic.set_dtt 'yes'
-      end
+    DevelopersTopic.set_dtt "in progress (@#{@dotd_name})"
+    pr_number = GitHub.create_and_merge_pull_request(
+      base: 'test',
+      head: 'staging',
+      title: 'DTT (Staging > Test)'
+    )
+    url = GitHub.url(pr_number)
+    wait_for "DTT (#{url}) to complete, re-run / investigate failures"
 
-      should_i 'update test room status with green commit and timestamp' do
-        new_sha = GitHub.sha 'test'
-        InfraTestTopic.set_green_commit new_sha
-      end
-    end
+    DevelopersTopic.set_dtt 'yes'
+
+    new_sha = GitHub.sha 'test'
+    InfraTestTopic.set_green_commit new_sha
```

Comment:
It seems a little aggressive to call the commit green just because I pressed enter after seeing `wait_for` "DTT (#{url}) to complete, re-run / investigate failures". How about replacing the aforementioned `wait_for` with `should_i 'update test room status with green commit and timestamp'`? That would leave you with the same number of prompts and hopefully a bit more clarity as to what is happening.